packages/graphql-language-service-server/src/MessageProcessor.ts

```diff
@@ -89,58 +89,48 @@ function toPosition(position: VscodePosition): IPosition {
   return new Position(position.line, position.character);
 }
 
-export class MessageProcessor {
+class WorkspaceMessageProcessor {
```

Comment:
Brilliant! An elegant solution

Reply:
Would you be interested in separating the `WorkspaceMessageProcessor` into a separate file? the less cognitive overhead, the more useful it is as a reference implementation, the more accessible it is to contributors!

Reply:
Thank you for your review!
Separated to new file.